braze/src/main/java/com/tealium/remotecommands/braze/BrazeTracker.java

```diff
@@ -632,4 +640,39 @@ private AppboyUser getAppboyUser() {
     private Appboy getAppboyInstance() {
         return Appboy.getInstance(mTealiumConfig.getApplication().getApplicationContext());
     }
+
+    @Override
+    public void onActivityCreated(Activity activity, Bundle savedInstanceState) {
+        mCurrentActivity = activity;
+    }
+
+    @Override
+    public void onActivityStarted(Activity activity) {
+        mCurrentActivity = activity;
+    }
+
+    @Override
+    public void onActivityResumed(Activity activity) {
+        mCurrentActivity = activity;
+    }
+
+    @Override
+    public void onActivityPaused(Activity activity) {
```

Comment:
Any reason we need these empty methods?

Reply:
yes - they're part of implementing `Application.ActivityLifecycleCallbacks`